src/App.js

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect } from "react";
+import { getTodos, setSearchText, setTodosList } from "./actions";
+import { connect } from "react-redux";
+import "./App.css";
+import { DataTable, Paginator, SearchField } from "lucid-ui";
+import { actionTypes } from "./actions/actionTypes";
+
+const {
+  EmptyStateWrapper,
+  EmptyStateWrapper: { Title, Body },
+} = DataTable;
+
+const App = ({
+  fetchTodos,
+  setPage,
+  setPageSize,
+  setTodosList,
+  page,
+  todos,
+  page_size,
+  todos_list,
+  error,
+  search_text,
+  setSearch,
+  fetchSearchedTodos,
+}) => {
+  const columns = [
```

Comment:
If this is a const why to put within a component. Extract this from the component